tools/locale.mjs

```diff
@@ -0,0 +1,78 @@
+import { readFileSync, writeFileSync } from "fs";
+import { dirname, join } from "path";
+import { fileURLToPath } from "url";
+import glob from "glob";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const newLocale = process.env.npm_config_new;
+const localeToUpdate = process.env.npm_config_update_locale;
+const updateAllLocales = process.env.npm_config_update_locales;
+
+const sortKeys = (content) => {
+  const contentJSON =
+    typeof content === "string" ? JSON.parse(content) : content;
+  const orderedResults = Object.keys(contentJSON)
+    .sort()
+    .reduce(function (result, key) {
+      result[key] = contentJSON[key];
+      return result;
+    }, {});
+
+  return JSON.stringify(orderedResults, null, "  ");
+};
+
+const packageNls = readFileSync(join(__dirname, "..", "package.nls.json"));
```

Comment:
nit: it might be a good idea to have something like
```
const packagePath = join(__dirname, "..");
const l10nPath = join(__dirname, "..", "l10n");
```
so that the directory is managed at single place.